### DISCUSSION TITLE
Do not show subnote card

### DISCUSSION THREAD

**Discussion by firebone-be** [TriliumNext/Trilium#3712] [Q&A]
Is it possible to not show a subnote card in the parent node?

**Comment by dumblegore** [comment]
yes, by using the "archive" attribute.

**Reply by firebone-be** [reply]
And if the subnote doesn't need to be archived?

**Reply by XXXJetfireXXX** [reply]
Oh, I think OP meant that he wants to hide these at the end of a note:
![image](https://user-images.githubusercontent.com/43646178/225296474-c9574ef6-e38f-4f31-84ba-1207f375ae11.png)

To do so, use the attribute "#hideChildrenOverview"

**Reply by firebone-be** [reply]
That is correct @XXXJetfireXXX, Thank you!
